djsixpack/models.py

```diff
@@ -1 +1,14 @@
-# No models
+from django.db import models
+
+
+class SixpackParticipant(models.Model):
+    """
+    A model that contains fields relating to a sixpack experiment participant.
```

Comment:
Can you either make a more fulsome docstring (what do each field mean?), or drop this.